test/integration/clientset_test.go

```diff
@@ -164,11 +169,11 @@ func TestBrokerClient(t *testing.T) {
 
 	brokerServer.Spec.AuthSecret = authSecret
 
-	brokerUpdated, err := brokerClient.Update(brokerServer)
+	updated, err := client.Update(brokerServer)
```

Comment:
I favor leaving broker in the variable name. To be explicit. Is there a refactor you intend to land on top of this that will test all resources with one script? If not, I do not understand why we would change this to be vague.